actors/market/src/state.rs

```diff
@@ -83,16 +83,12 @@ pub struct State {
     /// or has data replaced.
     /// Grouping by provider limits the cost of operations in the expected use case
     /// of multiple sectors all belonging to the same provider.
-    /// HAMT[ActorID]HAMT[SectorNumber]SectorDealIDs
+    /// HAMT[ActorID]HAMT[SectorNumber][]DealID
     pub provider_sectors: Cid,
 }
 
 /// IDs of deals associated with a single sector.
-#[derive(Clone, Debug, PartialEq, Eq, Serialize_tuple, Deserialize_tuple)]
-#[serde(transparent)]
-pub struct SectorDealIDs {
-    pub deals: Vec<DealID>,
-}
+pub type SectorDealIDs = Vec<DealID>;
```

Comment:
Do we even need the alias now?

Reply:
This may require a migration, previously the wire format was `[[1,2,3,4]]`. With this change, the format will be `[1,2,3,4]`.

Reply:
I don't think this code is live on chain (and it currently specifies `#[serde(transparent)]`.

Reply:
Ahh, right, then yeah, let's go for it.

Reply:
Remove the alias since it's unused.